providers/base/units/watchdog/test-plan.pxu

```diff
@@ -7,13 +7,6 @@ nested_part:
     watchdog-manual
     watchdog-automated
 
-id: watchdog-manual
-unit: test plan
-_name: Manual watchdog tests
-_description: Manual watchdog tests for Ubuntu Core devices
-include:
-    watchdog/trigger-system-reset
```

Comment:
I think you can just delete the last line (`watchdog/trigger-system-reset`) and keep the rest, just in case we need to write other manual Watchdog tests in the future. (most nested test plans have a `-manual` and an `-automated` counterparts, it's better to keep it this way)